tests/ui/dis/target_features_err.rs

```diff
@@ -0,0 +1,10 @@
+// build-fail
+// compile-flags: -Ctarget-feature=+rayTracingKHR,+ext:SPV_KHR_ray_tracing
+
+use spirv_std as _;
+
+#[spirv(any_hit)]
+pub fn main() {
+    unsafe { spirv_std::arch::terminate_ray() }
+}
+
```

Comment:
Feels kind of weird to have this in the `dis` folder since it's not actually a disassembly test, but shrug, doesn't matter too much (we should probably reorganize each file in `dis` to be in its actual proper place, I kind of just slapped everything here and said "let's deal with it later")